src/components/panel/demoGroups/index.html

```diff
@@ -0,0 +1,44 @@
+<div ng-controller="PanelGroupsCtrl as ctrl" ng-cloak>
+
+  <md-toolbar class="md-accent">
+    <div class="md-toolbar-tools">
+      <md-button
+        class="md-icon-button"
+        aria-label="Settings"
+        ng-click="ctrl.showMenu($event, 'settings', 'toolbar')">
```

Comment:
Use constants from the controller instead of strings for 'settings', 'favorite', and 'more'. It's a better code example. (More robust, less likely for typos or errors.)

Also since all of these are "toolbar" groups, I'd expect a more common use case is to declare the group name directly in the open function rather then passing it in. I would think when a user calls open, it's mapped to one method that knows what group it should be.